src/Riverline/MultiPartParser/Part.php

```diff
@@ -46,7 +46,7 @@ public function __construct($content)
     protected function parseContentString($content)
     {
         // Split headers and body
-        $splits = preg_split('/(\r?\n){2}/', $content, 2);
```

Comment:
@kriver, can you create a pull-request with the fix and a test case ?

@nazar-pc, we need to keep the current regex parsing to ensure BC. 

Thanks !